testdata/expected/docker/java-gradle-sample/com.sourcegraph/java-gradle-sample/JavaArtifact.graph.json

```diff
@@ -866,38 +899,58 @@
       "End": 323
     },
     {
+      "DefUnitType": "JavaArtifact",
+      "DefUnit": "com.sourcegraph/java-gradle-sample",
       "DefPath": "mypkg/FooClass:type/myIntegers",
       "File": "src/main/java/mypkg/FooClass.java",
       "Start": 378,
       "End": 388
     },
     {
-      "DefPath": "mypkg/FooClass:type/myInt",
+      "DefUnitType": "JavaArtifact",
+      "DefUnit": "com.sourcegraph/java-gradle-sample",
+      "DefPath": "mypkg/FooClass:type/myStatic",
       "Def": true,
       "File": "src/main/java/mypkg/FooClass.java",
-      "Start": 521,
-      "End": 526
+      "Start": 179,
```

Comment:
These reflect changes to `github.com/sgtest/java-gradle-sample`, which should be merged into that repository concurrently with this PR, right?